airavata_custos_portal/settings.py

```diff
@@ -0,0 +1,156 @@
+"""
+Django settings for airavata_custos_portal project.
+
+Generated by 'django-admin startproject' using Django 3.2.16.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+import json
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get(
+    "DJANGO_SECRET_KEY",
+    "django-insecure-=3_8+_&j54+-&(j(s2v=p#7%#$)oihyl)rdmp8(^5#r3y2wbs)",
+)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True #os.environ.get("DJANGO_DEBUG", "true") == "true"
```

Comment:
I think the environment variable should be kept. In production we'll want to turn off DEBUG.